constraints.txt

```diff
@@ -69,7 +69,6 @@ parasail==1.2.4
 Pillow==8.4.0
 pkg_resources==0.0.0
 pluggy==1.0.0
-Polygon3==3.0.8
```

Comment:
Any chance to add `Shapely` to constraints?

Reply:
It is already here 🙂